icefoganalytics/elcc-data-management
# Migrate the `logs` Table From Knex to Sequelize

Fixes https://github.com/icefoganalytics/elcc-data-management/issues/29

# Context

This PR covers migrating the `log table to a Sequelize backed models

# Testing Instructions

1. Check that the code complies via `dev up`.
2. Check that you can still log in at http://localhost:8080/.
